packages/typescript-estree/tests/lib/parse.test.ts

```diff
@@ -367,6 +367,22 @@ describe('parseAndGenerateServices', () => {
             alignErrorPath(error as Error);
           }
         };
+      const testParseExperimentalProjectService =
+        (filePath: string, extraFileExtensions: string[] = ['.vue']) =>
```

Comment:
[Testing] A couple of thoughts:

```suggestion
      const testExtraFileExtensions =
        (filePath: string, extraFileExtensions: string[]) =>
```

* This test is basically only used for `extraFileExtensions`, so let's name it like that
* IMO it's nice to be a bit more explicit on what settings are actually being tested

Reply:
Followed your suggestion. Just want to highlight that this is specifically testing extraFileExtensions when projectService is true, and that there are other tests to cover extraFileExtensions when projectService is not used.